homeassistant/components/nanoleaf/light.py

```diff
@@ -176,61 +143,43 @@ async def async_turn_on(self, **kwargs):
 
         if hs_color:
             hue, saturation = hs_color
-            await self._light.set_hue(int(hue))
-            await self._light.set_saturation(int(saturation))
+            await self._nanoleaf.set_hue(int(hue))
+            await self._nanoleaf.set_saturation(int(saturation))
```

Comment:
Odd that it doesn't allow setting both at once

Reply:
If you put both in the api call at the same time will the device accept it?

https://github.com/milanmeu/aionanoleaf/blob/master/aionanoleaf/nanoleaf.py#L229

Reply:
Ideally `async_turn_on` would build up a dict/set of commands and make a single api call